test/integration/nicotine.robot

```diff
@@ -11,5 +11,5 @@ Library           nicotine_library.py
 
 Run nicotine as shell command
     Running nicotine starts a process    nicotine    3
-    Result should be    ${True}
+    Result should be    ${0}
```

Comment:
Hey @mathiascode. This was originally my proposed fix too the last time we had a similar issue in #122. See what @lene said [here](https://github.com/Nicotine-Plus/nicotine-plus/pull/122#issuecomment-633637570).

Reply:
I changed the approach a bit, since I didn't get Popen to exit on failure. We start Nicotine+ using subprocess.call with a timeout. If the program is still running when the timeout is reached, the test was successful, and the result is set to True.